tests/network/hotplug.go

```diff
@@ -0,0 +1,161 @@
+/*
+ * This file is part of the kubevirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package network
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	expect "github.com/google/goexpect"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/libvmi"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+var _ = SIGDescribe("nic-hotplug", func() {
+	var virtClient kubecli.KubevirtClient
+
+	BeforeEach(func() {
+		var err error
+
+		virtClient, err = kubecli.GetKubevirtClient()
+		util.PanicOnError(err)
+	})
+
+	Context("a running VMI", func() {
+		const (
+			bridgeName  = "supadupabr"
+			ifaceName   = "iface1"
+			networkName = "skynet"
+			vmIfaceName = "eth1"
+		)
+
+		var vmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			vmi = setupVMI(virtClient, libvmi.NewAlpineWithTestTooling(libvmi.WithMasqueradeNetworking()...))
```

Comment:
Seems you are using methods defined in other test files.
While this may be possible technically, it introduces inter-test dependencies which we do not want.

You will need a commit that places these common helpers (or variables/constants) at a lib and then re-used them, not in this implicit way.

Reply:
I agree